#inputs
bool setBlowerState
---
#outputs
#bool isBlowerOn